### ISSUE TITLE
Add the output of the passed or failed tests

### ISSUE THREAD

**Issue by RoyFlo360** [prashanth-sams/pytest-html-reporter#202]
Print the logs of the passed or failed tests, it should be generic for both the passed and failed tests.
Or differentiate if we want to print only the passed or failed logs

**Comment by srinat999** [comment]
Would really like to have this feature. We are evaluating this for generating reports for our API Tests and to be able to display the requests and responses would be really appreciated. Something similar to what pytest-html does for capturing stdout would be enough.

**Comment by login4ajeet** [comment]
Yes logs/output statement should be there for both passed and failed tests.

**Comment by prashanth-sams** [comment]
@srinat999 @RoyFlo360 @login4ajeet 
Added in version 0.3.3 along with the other new features. I recommend you to take the latest build as I might push new versions.

<img width="1689" height="848" alt="Image" src="https://github.com/user-attachments/assets/d9eb309e-250f-4221-98db-34b86cf51e00" />
